packages/website/next.config.js

```diff
@@ -7,7 +7,7 @@ const withCSS = require('@zeit/next-css');
 const handleConfig = require('./bin/handle-config');
 
 const configPath = process.env.DOCS_WEBSITE_CONFIG_PATH;
-const cwd = process.env.DOCS_WEBSITE_CWD;
+const cwd = process.env.DOCS_WEBSITE_CWD || process.cwd();
```

Comment:
I know I was the one who added this for you, but I was thinking perhaps we should change this to instead throw an error if `process.env.DOCS_WEBSITE_CWD` is missing. A loud error message is better than things breaking in mysterious ways.